src/ipc-handlers.ts

```diff
@@ -119,11 +120,16 @@ export async function importSite(
 			sendIpcEventToRendererWithWindow( parentWindow, 'on-import', data, id );
 		};
 		const result = await importBackup( backupFile, site.details, onEvent, defaultImporterOptions );
+
+		bumpStat( STATS_GROUP.STUDIO_IMPORT, STATS_METRIC.SUCCESS );
+		bumpStat( STATS_GROUP.STUDIO_IMPORT_TYPE, backupFile.type );
```

Comment:
I believe backupFile.type will indicate whether the file is a zip, SQL, or tar.gz format. However, it might be more interesting to differentiate among jetpack, playground, local, etc.